libs/feature/catalog/src/lib/sources/sources.model.ts

```diff
@@ -0,0 +1,26 @@
+export interface CatalogSource {
+  label: CatalogSourceLabel
+  name: string
+  type: string
+  uuid: string
+}
+
+export interface CatalogSourceLabel {
```

Comment:
Bof
You should have somewhere a type like `export type Iso3Langs = 'fre' | 'eng' |` ...
Then `label` is a `Record<Iso3Langs, string>`